include/tinycrypt/ecc.h

```diff
@@ -85,7 +85,7 @@ extern "C" {
 /* Return values for functions, chosen with large Hamming distances between
  * them (especially to SUCESS) to mitigate the impact of fault injection
  * attacks flipping a low number of bits. */
-#define UECC_SUCCESS            0x00FFAAAA
+#define UECC_SUCCESS            0xCC
 #define UECC_FAILURE            0x75555555
 #define UECC_FAULT_DETECTED     0x7aaaaaaa
```

Comment:
@AndrzejKurek , will it be helpfull to make smaller also UECC_FAILURE and UECC_FAULT_DETECTED values ?

Reply:
Changing both of them gives us additional 108 bytes reduction.